src/components/rating/Star.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Icon, {types, colors} from '../icons/Icon';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+
+const Star = ({size, checked, onClick = () => undefined}) => {
```

Comment:
ok :) 
But in Star we need it. I could delete dump function and make it required and change tests.
Star is integral part of Rating